### ISSUE TITLE
No restriction on messsage handlers in insufficient-postmessage-origin-validation

### ISSUE THREAD

**Issue by Sjord** [returntocorp/sgrep-rules#1359]
[The insufficient-postmessage-origin-validation rule](https://github.com/returntocorp/semgrep-rules/blob/184c732f668f682b132bad76c4d29371fb9c0671/javascript/browser/security/insufficient-postmessage-origin-validation.yaml) matches any window.addEventListener. This causes false positives. E.g. it matches on the following, which I would not expect.

```
window.addEventListener("load", () => { console.log("app is finished loading"); });
```

I expect it to only match "message" events.

**Comment by Sjord** [comment]
Fixed with https://github.com/returntocorp/semgrep-rules/pull/1415
